s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3WireMockBase.scala

```diff
@@ -35,19 +36,35 @@ abstract class S3WireMockBase(_system: ActorSystem, _wireMockServer: WireMockSer
 
   val body = "<response>Some content</response>"
   val bucketKey = "testKey"
+  val bucketKeyForRanged = "testKeyForRanged"
```

Comment:
taking into account that we use Wiremock and that `bucketKey` is used as a part of request URI, it would be hard to tweak Wiremock to return different mocked response for the same endpoint based just on presence of `Range` header